<launch>



	<node pkg="main_loop" type="agent_new_3" name="agent_new_3" respawn="true" >
	</node>

	<node pkg="eurobot_gui" type="eurobot_gui.py" name="eurobot_gui" output="screen" respawn="true" >
	</node>
	
	
	   <include file="$(find lidar_2020)/launch/octagonal_range.launch">
    </include>	
	
    


</launch>
